test/helpers/erc7579-modules.js

```diff
@@ -0,0 +1,17 @@
+const SocialRecoveryExecutorHelper = {
+  RecoveryStatus: {
+    NotStarted: 0,
+    Started: 1,
+    Cancelled: 2,
+  },
```

Comment:
Using `const {Enum } = require("lib/@openzeppelin-contracts/test/helpers/enums");` you can do 

```suggestion
RecoveryStatus: Enum('NotStarted', 'Started', 'Cancelled');
```